apollo-ios-codegen/Sources/GraphQLCompiler/JavaScript/rollup.config.js

```diff
@@ -22,7 +22,7 @@ const options = {
     }),
     replace({
       preventAssignment: true,
-      "process.env.NODE_ENV": JSON.stringify("production"),
+      "globalThis.process.env.NODE_ENV": JSON.stringify("production"),
```

Comment:
This was causing an error but it is also incorrect because had there not been any error it would have resulted in `globalThis.production` which is not valid. I don't have the full context for our rollup configuration but based on what we're replacing we're effectively saying "we don't care what the environment is, always behave as if it's production". This gets us back to that with the latest code in graphl-js which [changed between `alpha.2` and `alpha.3`](https://github.com/graphql/graphql-js/commit/61025233e2123b5d3a4b729f5f329706bab19a70).

Note that replace wouldn't have worked with the alpha.2 code anyways but I believe the intention that we always want to operate in a "production" environment is correct for our use of the JS code.